### PR TITLE
Reenables smartgunner

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Reenables smartgunner, sg stays useable by anyone.

## Why It's Good For The Game
This was not a good idea overall, marines needed smartgunners in the early game to be able to push overall. making the SG useable by anyone is a good idea however.

## Changelog
:cl:
balance:marine smartgunners are back, sg crate in req only gives the gun again. sg is still useable by aynone
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -889,6 +889,7 @@
 	gun_iff_signal = list(ACCESS_IFF_MARINE)
 	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_AUTOMATIC)
 	starting_attachment_types = list(/obj/item/attachable/stock/t29stock, /obj/item/attachable/t29barrel)
+	gun_skill_category = GUN_SKILL_SMARTGUN //Uses SG skill for the penalties.
```
Comment (Pariah919) [inline code/modules/projectiles/guns/rifles.dm:892]:
```suggestion
```

File: code/modules/projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -862,7 +862,7 @@
 
 /obj/item/weapon/gun/rifle/standard_smartmachinegun
 	name = "\improper T-29 smart machine gun"
-	desc = "The T-29 is the TGMC's current standard IFF-capable medium machine gun. It's known for its ability to lay down heavy fire support very well. It is generally used when someone wants to hold a position or provide fire support. It cannot turn off IFF. It uses 10x26mm ammunition."
+	desc = "The T-29 is the TGMC's current standard IFF-capable medium machine gun. It's known for its ability to lay down heavy fire support very well. It is generally used when someone wants to hold a position or provide fire support. Requires special training and it cannot turn off IFF. It uses 10x26mm ammunition."
```
Comment (Pariah919) [inline code/modules/projectiles/guns/rifles.dm:865]:
```suggestion
	desc = "The T-29 is the TGMC's current standard IFF-capable medium machine gun. It's known for its ability to lay down heavy fire support very well. It is generally used when someone wants to hold a position or provide fire support. It cannot turn off IFF. It uses 10x26mm ammunition."
```

**Comment by Walarks** [comment]
Why not buff aim mode?

**Comment by SplinterGP** [comment]
> Why not buff aim mode?

Aimmode sucks.

**Comment by ilikesaying** [comment]
~~why not give aimmode to lasgun~~

**Comment by SpaceLoveSs13** [comment]
FINALLY!

**Comment by TiviPlus** [comment]
I agree but it's not next week yet

**Comment by neohenry** [comment]
Told you all.

**Comment by DerrickChamberlain** [comment]
How much do you want to bet they will push it to the right again. Next week, next week, next week, next week. How many more weeks do we need to wait? Put this in democracy and hold a vote for the community.

**Comment by DerrickChamberlain** [comment]
#bringbackSGs, #4SGs was good enough

**Pull request closed without merging**